### PR TITLE
Add SerialColumn

### PR DESCRIPTION
**Pull request by goke-epapa**
This is an improvement to provide SerialColumn functionality as the Yii2 SerialColumn doesn't. See  issue https://github.com/yii2tech/csv-grid/issues/2.

### CODE REVIEW

File: SerialColumn.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     */
+    public $header = '#';
+
+
+    /**
+     * @inheritdoc
+     */
+    public function renderHeaderCellContent()
+    {
+        return parent::renderHeaderCellContent();
```
Comment (klimov-paul) [inline SerialColumn.php:24]: This code is redundant: what the sense of redeclaring method just to call for its parent implementation?

File: SerialColumn.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function renderFooterCellContent()
+    {
+        return parent::renderFooterCellContent();
```
Comment (klimov-paul) [inline SerialColumn.php:32]: Same here.

File: SerialColumn.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function renderDataCellContent($model, $key, $index)
+    {
+        $pagination = $this->grid->dataProvider->getPagination();
```
Comment (klimov-paul) [inline SerialColumn.php:40]: This logic is incorrect: it will fail in case `CsvGrid` is specified with the `Query` which support `batch()`.

**Comment by klimov-paul** [comment]
Closed, since there is no response from the author.

**Comment by goke-epapa** [comment]
Hi @klimov-paul, 

I sincerely apologize for my lack of feedback. I see you have implemented the SerialColumn now.

I really hoped I would get around to replying and fixing the issues mentioned 😞, but for some reason I couldn't get to.

Thank you for your awesome Yii2 extension, I hope I can contribute sometime in the future.

**Pull request closed without merging**